### PR TITLE
Upgrade to v5 and add example.js

### PR DESCRIPTION
**Pull request by emilbayes**
Depends on https://github.com/yoshuawuyts/choo/pull/493 though

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ function redirect (url, rootSelector) {
   assert.equal(typeof url, 'string')
   assert.equal(typeof rootSelector, 'string')
 
-  return function redirectView (state, prev, send) {
-    send('location:set', url)
+  return function redirectView (state, emit) {
+    emit('pushState', url)
```
Comment (yoshuawuyts) [inline index.js]: This should become `replaceState` after https://github.com/yoshuawuyts/choo/pull/494 right?

Comment (emilbayes) [reply index.js]: In my opinion, yes :)

**Comment by emilbayes** [comment]
Should update `choo` in `devDependencies` when the new feature is released

**Review by yoshuawuyts — APPROVED** [review]
Yay, thanks!

**Comment by yoshuawuyts** [comment]
@emilbayes semver should take care of that I think, this good enough (:

**Comment by yoshuawuyts** [comment]
`v2.0.0` :tada:

**Pull request merged**